core/modules/admin_bar/css/admin_bar.css

```diff
@@ -127,9 +129,14 @@
 #admin-bar .dropdown li > a,
 #admin-bar .dropdown li > span {
   display: block;
-  padding: 0.75em 1.5em 0.75em 1em;
+  padding: 0.5em 1.5em 0.5em 0.75em;
+}
+#admin-bar-icon > li.expandable > a,
```

Comment:
An expandable first level link ends up with different padding then a regular first level link. Is this on purpose?

Reply:
Hm, you may be right, that might not look good. :thinking:
Do you have any example for such a "standalone" link?

Reply:
I can't seem to figure out a way to introduce a link that's _not_ child of "expandable". For example, if I place a link in the administration menu, it's child of an expandable ul.
Am I missing something?

Reply:
I realized that too after I mentioned it. I was assuming by the css that it was possible. When does the original css work then? It seemed to be overridden by the one you added.

Reply:
The original above works for sublevels, which need some extra padding for (potential) arrows, the more specific one here, that overrides the generic rule above, is for the top level. The one with the icons, that never has those arrows, and where that extra space would just be wasted.